Common/src/main/java/com/blamejared/crafttweaker/impl/service/BridgeService.java

```diff
@@ -0,0 +1,15 @@
+package com.blamejared.crafttweaker.impl.service;
+
+import com.blamejared.crafttweaker.api.ICraftTweakerRegistry;
+import com.blamejared.crafttweaker.impl.registry.CraftTweakerRegistry;
+import com.blamejared.crafttweaker.platform.services.IBridgeService;
+
+public final class BridgeService implements IBridgeService {
```

Comment:
Is this a service for a single thing?

Not against it, just want to know why

Reply:
WIP; if needed I'll add other stuff here